Description: replace ava by tape in test
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-04-29

--- a/test.js
+++ b/test.js
@@ -1,5 +1,5 @@
-import test from 'ava';
-import cliTruncate from '.';
+const test = require('tape');
+const cliTruncate = require('.');
 
 test('main', t => {
 	t.is(cliTruncate('unicorn', 4), 'uni…');
@@ -19,6 +19,7 @@
 	t.is(cliTruncate('unicorn', 6, {position: 'start'}), '…icorn');
 	t.is(cliTruncate('unicorn', 5, {position: 'middle'}), 'un…rn');
 	t.is(cliTruncate('unicorns', 6, {position: 'middle'}), 'uni…ns');
+    t.end();
 });
 
 test('space option', t => {
@@ -33,6 +34,7 @@
 	t.is(cliTruncate('\u001B[31municornsareawesome\u001B[39m', 10, {position: 'middle', space: true}), '\u001B[31munico\u001B[39m … \u001B[31mme\u001B[39m');
 	t.is(cliTruncate('Plant a tree every day.', 14, {position: 'middle', space: true}), 'Plant a … day.');
 	t.is(cliTruncate('안녕하세요', 4, {position: 'start', space: true}), '… 요', 'wide char');
+    t.end();
 });
 
 test('preferTruncationOnSpace option', t => {
@@ -42,4 +44,5 @@
 	t.is(cliTruncate('unicorns rainbow dragons', 6, {position: 'end', preferTruncationOnSpace: true}), 'unico…');
 	t.is(cliTruncate('unicorns rainbow dragons', 6, {position: 'middle', preferTruncationOnSpace: true}), 'uni…ns');
 	t.is(cliTruncate('unicorns partying with dragons', 20, {position: 'middle', preferTruncationOnSpace: true}), 'unicorns…dragons');
+    t.end();
 });
